include/ada/ada_idna.h

```diff
@@ -88,7 +88,7 @@ namespace ada::idna {
 /**
  * @see https://www.unicode.org/reports/tr46/#Validity_Criteria
  */
-bool is_label_valid(const std::u32string_view label);
+bool is_label_valid(std::u32string_view label);
```

Comment:
ada_idna changes needs to be made to ada-url/idna repository. can you open a pull request to it as well?

Reply:
You mean this right here?
https://github.com/ada-url/idna/blob/3c8be01d42b75649f1ac9b697d0ef757eebfe667/include/ada/idna/validity.h#L12

Why are there 2 repos?

Reply:
To distribute separately

Reply:
https://github.com/ada-url/idna/pull/38